src/MicroflowTimer/MicroflowTimer.xml

```diff
@@ -24,12 +20,27 @@
             <category>Behavior</category>
             <description>If true (and execute once is false), the microflow will be invoked the first time if the widget has loaded. If false, the microflow will be invoked the first time after interval has passed.</description>
         </property>
-        <property key="microflow" type="microflow" required="true">
+        <property key="callEvent" type="enumeration" defaultValue="callMicroflow">
+            <caption>Call event</caption>
+            <category>Behavior</category>
+            <description/>
+            <enumerationValues>
```

Comment:
I suggest another option ` <enumerationValue key="doNothing">Do nothing</enumerationValue>`

Reply:
Widget requires a microflow or nanoflow at all times since that's its major functionality.